### PR TITLE
Final Project First Draft

### PR DESCRIPTION
**Pull request by LindsayTaylor**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+
+#container{
+  display: none;
+  max-width: 960px;
+  margin: auto;
+}
+
+body{
+  background: url("space4.gif");
```
Comment (lukeruth) [inline css/style.css:14]: Try cleaning up your folder structure and using an images folder.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+#page-header{
+  border:0;
+  box-sizing: border-box;
+  clear: both;
+  display: block;
+  float: left;
+  margin: auto;
+  padding: 2% 8% 2% 8%;
```
Comment (lukeruth) [inline css/style.css:24]: If the first two values are the same as the second two, you can omit them. So it would look like `padding: 2% 8%`

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  width: 100%;
+}
+
+.clearfix{
```
Comment (lukeruth) [inline css/style.css:28]: Clearfix is generally understood to take a very specific action, which is actually more than just clearing the floats. The CSS for this class should be:

```
.clearfix:after {
  content: "";
  display: table;
  clear: both;
}
```

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  padding: 1%;
+}
+
+.nav li:hover{
+  background: white;
+}
+
+.nav li a:hover{
+  color: black;
+}
```
Comment (lukeruth) [inline css/style.css:47]: This is a cool effect - using the inverse of the background and font color. Nice use of pseudo selectors!

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+.nav li a{
+  color: white;
+  font-size: 24px;
```
Comment (lukeruth) [inline css/style.css:51]: Now that we've learned about responsive font units - try converting these pixel fonts to something like ems or rems.

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+  list-style: none;
+  text-decoration: none;
+  width: 15%;
+}
+
+main{
+  box-sizing: border-box;
```
Comment (lukeruth) [inline css/style.css:58]: Rather than applying this to a single element, you should apply the same `box-sizing` to all elements on your page so they behave consistently. 

```
* {
 box-sizing: border-box;
}
```

File: index.html [hunk 7]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="description" content="This is a boilerplate for building websites." />
```
Comment (lukeruth) [inline index.html:5]: Fill in this description!

File: index.html [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+    <title>SPAcE DiNoSAuR</title>
```
Comment (lukeruth) [inline index.html:6]: This is an amazing band name. Are you part of it?!

File: index.html [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+    <link rel="stylesheet" href="css/reset.css" type="text/css" />
```
Comment (lukeruth) [inline index.html:8]: Good use of reset.css

File: index.html [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+
+          <main>
+            <div class="content">
+              <h1>SPAcE DiNoSAuR</h1>
+              <img src="http://placehold.it/450x700">
+            </div>
+          </main>
+          <footer>
+            <div id="social">
+  						<a href="facebook.com"><img class="media" src="facebook.png"></a>
```
Comment (lukeruth) [inline index.html:44]: Watch your indentation here - it's probably a mix of spaces and tabs. Either is fine just pick one and stick with it.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+  						<a href="twitter.com"><img class="media" src="twitter.png"></a>
+  					</div>
+          </footer>
+      </div>
+    </div>
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+  <script src="index.js"></script>
```
Comment (lukeruth) [inline index.html:52]: Just like with CSS and images, try making a javascript folder so you keep things a bit more organized.

File: index.js [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+  /*$(window).load(function(){
+     $('#imageID').fadeOut('slow',function(){
+           setTimeout(function(){
+              $(this).fadeOut('slow');*/
+
+              function practice (){
+
+              }
+
+              $("#container").fadeIn(7000);
```
Comment (lukeruth) [inline index.js:12]: This is a pretty entertaining effect. Band sites can definitely get away with slightly less conventional design. I would try shortening it though. There's a rule of thumb in animation on the web that however long you feel it should be - cut it in half, and then cut it in half again. When we work on something so closely it starts to feel faster, but to a new visitor to the site it feels quite long.

Try 2 or 3 seconds and see what you think.

File: index.js [hunk 13]
```diff
@@ -0,0 +1,14 @@
+$(document).ready(function(){
+
+  /*$(window).load(function(){
+     $('#imageID').fadeOut('slow',function(){
+           setTimeout(function(){
+              $(this).fadeOut('slow');*/
+
+              function practice (){
+
+              }
```
Comment (lukeruth) [inline index.js:10]: If this is a placeholder for later, that's cool - but if it's not try removing unused code so that it's clear it's not a mistake.

File: music.html [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+                  <li>
+                    <a href="updates.html">UPDATES</a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </header>
+
+          <main>
+            <iframe width="75%" height="100" scrolling="no" frameborder="no" src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/147393995&amp;auto_play=false&amp;hide_related=false&amp;show_comments=true&amp;show_user=true&amp;show_reposts=false&amp;visual=true"></iframe>
```
Comment (lukeruth) [inline music.html:37]: Cool use of the SoundCloud iframe to get some interaction in your site.

File: rubric.md [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+| Use classes to identify a type of element and IDs for identifying a specific unique element on the page |                                |                        |                          |
+| Select the appropriate tags to markup content                                                           |                                |                        |                          |
+| Apply fonts, color and styles to elements and the page                                                  |                                |                        |                          |
+| Demonstrate use of floats (or inline-block) and the box model                                           |                                |                        |                          |
+| Use JavaScript/jQuery to make pages interactive                                                         |                                |                        |                          |
+| TOTAL: ______/24                                                                                        |                                |                        |                          |
```
Comment (lukeruth) [inline rubric.md:30]: Overall nice job. There are some fundamentals to clean up but overall you've got a cool design idea and a chance to play around. Make sure to clean up any unused CSS, JS or HTML. If an element isn't being referenced or styled, ask yourself "Does it need to be here?" It'll make your site easier to maintain and update in the long run.

This is ready to merge whenever you are!

Comment (LindsayTaylor) [inline rubric.md:30]: Thanks, Luke!  I have made a terrible habit of adding things and forgetting to take them out when I don't need them. I've now had multiple times where I went to change something and because of the unnecessary styling, my changes don't do what they should.
